web/src/compose_state.ts

```diff
@@ -58,6 +59,14 @@ export function has_recipient_viewed_topic_resolved_banner(): boolean {
     return recipient_viewed_topic_resolved_banner;
 }
 
+export function set_recipient_guest_emails(guest_emails: string): void {
+    recipient_guest_emails = guest_emails;
+}
```

Comment:
Why are we storing this as a string? looking at the name, I would expect this to be a set of emails.

We might wanna also change the name a little bit to signify that we're using this data for the warning banner.

Reply:
> Why are we storing this as a string? looking at the name, I would expect this to be a set of emails.

Strings are easier to compare. Can also use `string []`.

Should we rename it to something like `recipient_guest_emails_for_warning_banner`?

Reply:
It's better to use the data structure that is more appropriate for what we're storing. The use case is limited to comparison right now, leading to a string feeling like a good choice, but that may not always be the case in future.